pyEpiabm/pyEpiabm/routine/file_population_config.py

```diff
@@ -84,8 +84,12 @@ def make_pop(input_file: str, random_seed: int = None, time: float = 0):
         current_cell = None
         # Iterate through lines (one per microcell)
         for _, line in input.iterrows():
+
+            # Converting from float to string
+            cell_id_csv = str(int(line["cell"]))
```

Comment:
Is it possible to specify the type of the columns when we read this csv file in with pandas? Or is there a reason for doing this explicit conversion instead which I've missed?

Reply:
The id is currently a float in the csv file, so if we were to take it as a string, it would become "1.0" instead of "1" (which we want). We can look to see if there are ways to convert differently

Reply:
In that case perhaps we could just convert float -> int -> str when we feed it into the `set_id` function? But also I would be happy for you to change the output format in the csv files from `print_population` etc, as these don't need to be floats (or I can do this) - given the new form of the ID there is no reason not to be consistent here

Reply:
I think it'd be best to stick with having a string input to the set_id function, and the output format in the csv files are currently strings I believe, as they have changed now. I don't quite understand - the id needs to be changed from a float to a string somewhere, so is it not ok to do it here?

Reply:
Ahh I see, I was thinking about the wrong csv! I think I am happy for this to stay as is then. I might have a slight preference for reading in the values as integers (as this is what they really are) and then just converting to string to append to the full id (with the cell etc). To do this, you set `dtype={"cell":int}` in the `read_csv` call (or even just `dtype=int` if all elements should be integers, which I think they should!

Reply:
Ah ok sure, I will make this change